sip-quickstart/assets/admin/admin-client.js

```diff
@@ -52,11 +51,14 @@ class AdminClient {
   }
 
   async fetchState() {
-    const response = await fetch(
-      `./check-status?token=${encodeURIComponent(this.token)}`
-    );
-    await this._handleResponse(response);
-    return response.json();
+    if (this.token) {
```

Comment:
Thank you! I think we'll get this working with headers in a future iteration, but I really appreciate all the work to make this hack work saner!